logtape/category.ts

```diff
@@ -0,0 +1,3 @@
+export type CategoryList = Readonly<string[]>;
+
+export type Category = string | CategoryList;
```

Comment:
Could you add TSDoc comments for those two types?

Reply:
done